ci/Dockerfile.ubuntu

```diff
@@ -50,12 +50,12 @@ RUN apt-get install -y \
     g++-4.8 \
   || /bin/true
 
-# Install the bigtable emulator and the bigtable command-line client.  They
-# are used in the integration tests.
+# Install the Cloud Bigtable emulator and the bigtable command-line client.
+# They are used in the integration tests.
 ARG GOPATH=/var/tmp/build/cbt
 WORKDIR ${GOPATH}
 RUN if grep -q 14.04 /etc/lsb-release; then \
-      echo "Skip cloud bigtable client install, go version too old in Ubuntu 14.04."; \
+      echo "Skipping cloud bigtable client install, Go version too old in Ubuntu 14.04."; \
```

Comment:
s/cloud bigtable client/cbt/ ?

or if you want to be more descriptive, "cbt CLI"?

Reply:
Done.